contracts/finance/VestingWallet.sol

```diff
@@ -43,11 +42,10 @@ contract VestingWallet is Context, Ownable2Step {
      * @dev Sets the sender as the initial owner, the beneficiary as the pending owner, the start timestamp and the
      * vesting duration of the vesting wallet.
      */
-    constructor(address beneficiaryAddress, uint64 startTimestamp, uint64 durationSeconds) payable Ownable(msg.sender) {
-        if (beneficiaryAddress == address(0)) {
+    constructor(address beneficiary, uint64 startTimestamp, uint64 durationSeconds) payable Ownable(beneficiary) {
+        if (beneficiary == address(0)) {
```

Comment:
maybe not in the PR, but we should remove that check here and move it to `Ownable`'s constructor

Reply:
I agree we should reconsider this. I opened an issue
https://github.com/OpenZeppelin/openzeppelin-contracts/issues/4511